src/test/java/org/apache/drillui/test/framework/pages/ProfilesPage.java

```diff
@@ -0,0 +1,16 @@
+package org.apache.drillui.test.framework.pages;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+public class ProfilesPage extends BasePage {
+  @FindBy (css = "tbody>tr[role='row']>td>a")
+   private List<WebElement> queryResult;
+
+public boolean isQueryProfileINList(String queryProfile){
+    for(WebElement i :queryResult) {if (i.getAttribute("href").contains(queryProfile)) return true; }
```

Comment:
Please align your code.
You need to setup your IDE:
- to automatically add Apache header at the start of new classes
- to use indent 2 instead of 4
There is a settings file that you can import in IDEA:
[settings jar](https://cwiki.apache.org/confluence/download/attachments/29687985/intellij-idea-settings.jar?version=1&modificationDate=1381928827000&api=v2)

Reply:
@agozhiy Settings changed